free-2.md

```diff
@@ -11,41 +11,81 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+There are lots of instances where presentation slides are used as graphic design tools. This may be because Photoshop is expensive and hard to learn, and slides give enough drawing and diagram options for particular use cases. 
+
+However, some of these designs are intended for reuse and often have patterns in them. Like ContextFree, perhaps there is a way to declaratively make these slides/images without having to manually edit the information every time. The next section will give some more examples of use cases.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL will reduce the time it takes to create templated presentations, especially for the specific case of color-coded lyric videos. While the current technologies used may be more intuitive because of the visual nature of the videos, proficiency in DSL will allow creators to make these videos and upload them faster. This can lead to more views and subscribers compared to the competition. 
+
+While a regular API may be equally suitable for the job described previously, the control required to generalize this process for other similar cases is easier by implementing a DSL. One example of this is the convention of scheduled posters that VTubers post weekly. The DSL would fit into a data pipeline where schedules on an online calendar may be converted into a list of events as text, and the text and background can then be consolidated by programming in the DSL. 
+
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I came up with this idea because I encounter these types of media often due to my interests. One VTuber I watch also complained about the time it takes to create schedule posters, and wish there was some way to simply convert calendar events into posters magically. This also relates to my first idea about multi-lingual subtitling in fansubbing communities. Sometimes the material is audio only, and translators create their own layouts in a presentation-like format and make a video with subtitles from scratch. 
```

Comment:
I like how you thought about this issue from another person's perspective, you know that somebody would benefit from it